pkg/resourceinterpreter/defaultinterpreter/default.go

```diff
@@ -44,6 +57,7 @@ func (e *DefaultInterpreter) HookEnabled(kind schema.GroupVersionKind, operation
 		if _, exist := e.replicaHandlers[kind]; exist {
 			return true
 		}
+		return e.checkScaleSubResourceExist(kind)
```

Comment:
The pattern should be if containsScaleResource(xxx) {return true}.

Don't suspend the log below, the one close to the end of this function.